README.md

```diff
@@ -3,106 +3,214 @@
-* You do not need to write any code for registration or definition of
-[cat-components](https://github.com/catberry/catberry/blob/7.1.2/docs/index.md#cat-components) or
-[stores](https://github.com/catberry/catberry/blob/7.1.2/docs/index.md#stores),
-Catberry discovers them automatically on startup
-* Fast and efficient server-side [progressive rendering engine](http://www.phpied.com/progressive-rendering-via-multiple-flushes/) based on
-[node.js streams](http://nodejs.org/api/stream.html#stream_api_for_stream_implementors)
-* [Dust](https://github.com/catberry/catberry-dust),
-[Handlebars](https://github.com/catberry/catberry-handlebars) and
-[Jade](https://github.com/catberry/catberry-jade) template engines
-* No Virtual DOM – you can use any other library with Catberry without conflicts
+## What the cat is that?
+
+Catberry was developed to help in creating ["isomorphic/Universal" Web applications](https://github.com/catberry/catberry/blob/8.0.0-dev/docs/index.md#isomorphicuniversal-applications).
+
+Long story short, isomorphic/universal applications are apps that use shared codebase in both server and browser environments and can render pages working as [Single Page Application](http://en.wikipedia.org/wiki/Single_Page_Application) in a browser and as a back-end application on the server using the same modules.
+
+## TLDR;
+
+Install [Catberry CLI](https://www.npmjs.com/package/catberry-cli) using following command:
+
+```bash
+npm install -g catberry-cli
+```
+
+Now you can create an empty project with [Handlebars](http://handlebarsjs.com/) support like this:
+
+```bash
+catberry init empty-handlebars
+```
+
+Or an example application that works using GitHub API:
+
+```bash
+catberry init example
+```
+
+Also, you can get a list of all templates:
+
+```bash
+catberry init ?
+```
+
+## Useful links
+
+* [Catberry Documentation](https://github.com/catberry/catberry/blob/8.0.0-dev/docs/index.md)
+* [Get Started Guide](https://github.com/catberry/catberry/blob/8.0.0-dev/docs/index.md#get-started)
+* [Officially supported plugins](https://github.com/catberry/catberry/blob/8.0.0-dev/docs/index.md#https://github.com/catberry/catberry/blob/%23221-es6-docs/docs/index.md#list-of-officially-supported-plugins)
+* [Catberry's homepage](http://catberry.org) and its [source code](https://github.com/catberry/catberry-homepage)
+* [Todo application](https://github.com/catberry/catberry-todomvc)
+* [Example application](https://github.com/catberry/catberry-cli/tree/master/templates/example)
+
+## Why should I use that?
+
+### Architecture
+
+* The entire architecture of the framework is built using the [Service Locator](https://github.com/catberry/catberry/blob/8.0.0-dev/docs/index.md#service-locator) pattern that helps managing module dependencies and [create plugins](https://github.com/catberry/catberry/). Also,   [Flux](https://github.com/catberry/catberry/blob/8.0.0-dev/docs/index.md#flux) is used as a data layer
```

Comment:
* The entire architecture of the framework is built using the [Service Locator](https://github.com/catberry/catberry/blob/8.0.0-dev/docs/index.md#service-locator) pattern--which helps to manage module dependencies and [create plugins](https://github.com/catberry/catberry/)--and [Flux](https://github.com/catberry/catberry/blob/8.0.0-dev/docs/index.md#flux), for the data layer.